cnv_germline_case_scattered_workflow.inputs.json

```diff
@@ -0,0 +1,38 @@
+{
+  "CNVGermlineCaseScatteredWorkflow.normal_bams": [
+    "gs://gatk-best-practices/cnv_germline_pipeline/SM-74NEG_20xy-downsampled.bam",
+    "gs://gatk-best-practices/cnv_germline_pipeline/SM-74P2T_20xy-downsampled.bam",
+    "gs://gatk-best-practices/cnv_germline_pipeline/SM-74P35_20xy-downsampled.bam"
+    ],
+  "CNVGermlineCaseScatteredWorkflow.normal_bais": [
+    "gs://gatk-best-practices/cnv_germline_pipeline/SM-74NEG_20xy-downsampled.bam.bai",
+    "gs://gatk-best-practices/cnv_germline_pipeline/SM-74P2T_20xy-downsampled.bam.bai",
+    "gs://gatk-best-practices/cnv_germline_pipeline/SM-74P35_20xy-downsampled.bam.bai"],
+  "CNVGermlineCaseScatteredWorkflow.contig_ploidy_model_tar": "gs://gatk-best-practices/cnv_germline_pipeline/wes-do-gc-contig-ploidy-model.tar.gz",
+  "CNVGermlineCaseScatteredWorkflow.gatk_docker": "broadinstitute/gatk:4.1.5.0",
+  "CNVGermlineCaseScatteredWorkflow.allosomal_contigs": ["X", "Y"],
+  "CNVGermlineCaseScatteredWorkflow.ref_copy_number_autosomal_contigs": 2,
+  "CNVGermlineCaseScatteredWorkflow.gcnv_disable_annealing": true,
```

Comment:
Note that these parameters were set to decrease runtime for testing purposes, but it is actually not what we use to run gCNV with. I would probably just go with the default parameters

Reply:
removed